java-parent-pom/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>net.openhft</groupId>
         <artifactId>root-parent-pom</artifactId>
-        <version>1.2.5</version>
+        <version>1.2.5-SNAPSHOT</version>
```

Comment:
Strange that this one was set to 1.2.4, that hasn't been released yet (I checked in Nexus)